lessons/specifics/mnesia.md

```diff
@@ -113,6 +113,13 @@ We define the columns using the atoms `:id`, `:name`, and `:job`. When we execut
  - `{:atomic, :ok}` if the function executes successfully
  - `{:aborted, Reason}` if the function failed
 
+In particular, if the table already exists, the reason will be of the form `{:already_exists, table}` so if we try to create this table a second time, we will get:
+
+```shell
```

Comment:
That page had the `shell` tag in the existing text so I re-used that. I'm happy to change everything to `elixir`.